configuration/config.go

```diff
@@ -0,0 +1,24 @@
+package configuration
+
+type Config struct {
+	Build string `yaml:"build"`
+	Run   string `yaml:"run"`
+	Rules []Rule `yaml:"rules"`
+}
+
+type Rule struct {
+	Name     string `yaml:"name"`
+	Regex    string `yaml:"regex"`
```

Comment:
I give an example, but I thing that is not that good.
What do you think about `Match`? Do you have any other suggestion?

Reply:
I left it as Regex for now, but I think Match could work